2nd-gen/callable-functions/functions/index.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {onCall, HttpsError} = require("firebase-functions/v2/https");
+const {getDatabase} = require("firebase-admin/database");
```

Comment:
great use of the modular admin SDK. I'm going to add a note in the devrel plan to make sure everyone does this.

Reply:
added